k8s/namespaces/fees-pay/fees-register-frontend/demo.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fees-register-frontend
+spec:
+  releaseName: fees-register-frontend
+  values:
+    nodejs:
+      image: hmctspublic.azurecr.io/fees-register/frontend:pr-401 #{"$imagepolicy": "flux-system:demo-fees-register-frontend"}
```

Comment:
imagetag pr-401 does not look correct, image policy will update correctly. 

I am assuming pr-401 is not the specific tag you require for demo?

Reply:
updated it to pr no 432 :: same is there  we are pointing in apps/fees-pay/fees-register-frontend/demo-image-policy.yaml